mcs/class/System.Core/System.IO.Pipes/NamedPipeServerStream.NotSupported.cs

```diff
@@ -0,0 +1,71 @@
+using Microsoft.Win32.SafeHandles;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Security.Permissions;
+
+namespace System.IO.Pipes
+{
+	public sealed partial class NamedPipeServerStream : PipeStream
+	{
+		public NamedPipeServerStream (string pipeName, PipeDirection direction, int maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, int inBufferSize, int outBufferSize, PipeSecurity pipeSecurity)
+			: base (PipeDirection.In, 0)
+		{
+			throw new PlatformNotSupportedException ();
+		}
+
+		public NamedPipeServerStream (string pipeName, PipeDirection direction, int maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, int inBufferSize, int outBufferSize, PipeSecurity pipeSecurity, HandleInheritability inheritability)
+			: base (PipeDirection.In, 0)
+		{
+			throw new PlatformNotSupportedException ();
+		}
+
+		public NamedPipeServerStream (string pipeName, PipeDirection direction, int maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, int inBufferSize, int outBufferSize, PipeSecurity pipeSecurity, HandleInheritability inheritability, PipeAccessRights additionalAccessRights)
+			: base (PipeDirection.In, 0)
+		{
+			throw new PlatformNotSupportedException ();
+		}
+
+		[SecurityPermission (SecurityAction.Demand, Flags = SecurityPermissionFlag.ControlPrincipal)]
```

Comment:
Please don't add SecurityPermission (they are dead concept)

Reply:
This one was there to match CoreFX "NamedPipeServerStream.Unix.cs" sources.